dht/dht.go

```diff
@@ -156,16 +184,18 @@ func NewServer(c *ServerConfig) (s *Server, err error) {
 			panic(err)
 		}
 	}()
-	go func() {
-		err := s.bootstrap()
-		if err != nil {
-			select {
-			case <-s.closed:
-			default:
-				log.Printf("error bootstrapping DHT: %s", err)
+	if !s.config.NoBootstrap {
```

Comment:
I will confess that, while I think disabling global router bootstrap is a useful option overall, I added it in haste because I wanted to work on test cases and wanted a way to make an isolated DHT on my PC without risk of connecting to the DHT and earning a bad name. :)

So yes, placing this logic somewhere more..logical would be good, as would making what it means more explicit, such as "NoGlobalBootstrap" or something.